gflownet/proxy/crystals/dave.py

```diff
@@ -1,49 +1,11 @@
-import os
-import re
-import sys
 from copy import deepcopy
 from pathlib import Path
 
-import git
 import torch
 from torchtyping import TensorType
 
 from gflownet.proxy.base import Proxy
 
-# gflownet/ repo root
-ROOT = Path(__file__).resolve().parent.parent.parent.parent
-# where to clone / find the external code
-REPO_PATH = ROOT / "external" / "repos" / "ActiveLearningMaterials"
-# remote repo url to clone from
-REPO_URL = "https://github.com/sh-divya/ActiveLearningMaterials.git"
```

Comment:
I would actually keep this constant (and use it to construct `pip_url` in the body of the class).

Reply:
Not sure why you would allocate one more variable than needed but sure I can do that :)